server/graphql/v1/mutations/collectives.js

```diff
@@ -380,6 +380,42 @@ export function editCollective(_, args, req) {
 
   let collective, parentCollective;
 
+  const tiers = args.collective.tiers;
+  forEach(tiers, tier => {
+    const presets = tier.presets;
+    const amount = tier.amount;
+    const name = tier.name;
+
+    if (!tier.amountType) {
+      tier.amountType = presets ? 'FLEXIBLE' : 'FIXED';
+    }
+
+    const amountType = tier.amountType;
+    const minPreset = presets ? Math.min(...presets) : null;
+
+    if (!name || name.trim().length === 0) {
+      throw new Error('Name field is required for all tiers');
+    }
+
+    if (tier.type !== 'TICKET' && amountType === 'FIXED' && !amount) {
+      throw new Error(`In ${name}'s tier, "Amount" is required`);
+    }
+
+    if (amountType === 'FLEXIBLE' && !tier.minimumAmount) {
```

Comment:
`minimumAmount` is not required. If it was to address my previous comment, I was just asking to check that "minimum amount cannot be less than minimum suggested amounts" only if a value is provided.